app/models/admin.rb

```diff
@@ -0,0 +1,4 @@
+class Admin < ApplicationRecord
```

Comment:
No need for this model! Admin is a namespace rather than a resource. We will talk more about this on Tuesday! The general rule of thumb -- if there's something in a path that doesn't map to a resource in the database or a tangible piece of data, it is likely an organizational tool and can be used as a resource.